File: SISTEMA DE TICKET/Proyectoticket/Solicitante.cs
using System;

namespace Proyectoticket.logica
{
    public class Solicitante : Persona
    {
        public string StrDepartamento { get; set; }
        public string StrExtension { get; set; }

        public Solicitante(string strCodigo, string strNombre, string strCorreo, string strDepartamento, string strExtension)
            : base(strCodigo, strNombre, strCorreo)
        {
            this.StrDepartamento = strDepartamento;
            this.StrExtension = strExtension;
        }

        public override string ObtenerRol()
        {
            return "Solicitante";
        }

        public override void MostrarInformacion()
        {
            base.MostrarInformacion();
            Console.WriteLine($" Departamento: {StrDepartamento} | Extension: {StrExtension}");
        }
    }
}

File: SISTEMA DE TICKET/SISTEMA DE TICKET DIEGO/Solicitante.java
public class Solicitante extends Persona {
    private String strDepartamento;

    public Solicitante(String strNombre, String strCorreo, String strTelefono, String strDepartamento) {
        super(strNombre, strCorreo, strTelefono);
        this.strDepartamento = strDepartamento;
    }

    public String getStrDepartamento() { return strDepartamento; }
}